backend/cmmgen_state.ml

```diff
@@ -27,8 +27,24 @@ type ustructured_constant =
   | Const_int64 of int64
   | Const_nativeint of nativeint
   | Const_vec128 of
-      { high : int64;
-        low : int64
+      { word0 : int64;
```

Comment:
(Maybe duplicate the comment about
`word0` being the least significant?)

(If feels like the new names are slightly
worse, but I have unfortunately no
suggestions.)

Reply:
Added everywhere